packages/flutter/lib/src/rendering/stack.dart

```diff
@@ -8,6 +8,7 @@ import 'dart:math' as math;
 import 'dart:ui' show lerpDouble, hashValues;
 
 import 'package:flutter/foundation.dart';
+import 'package:vector_math/vector_math_64.dart';
```

Comment:
unused import?

Reply:
applyPaintTransform needs Matrix4,  change to: import 'package:vector_math/vector_math_64.dart' show Matrix4;